### PR TITLE
feat(docker-compose): bash history support (BCD-396)

### PR DESCRIPTION
**Pull request by uguy**
Add documentation for bash history support in BCD docker container.
(follow up of the previous PR https://github.com/bonitasoft/bonita-continuous-delivery-doc/pull/140 )

### CODE REVIEW

**Comment by JeremJR** [comment]
retest this please

**Comment by JeremJR** [comment]
retest this please

**Comment by JeremJR** [comment]
retest this please

**Comment by JeremJR** [comment]
retest this please

**Comment by tnguyen1** [comment]
retest this please

File: md/bcd_controller.md [hunk 1]
```diff
@@ -87,6 +87,20 @@ Then start the BCD controller container interactively with:
 $ docker-compose run --rm bcd
 ```
 
+#### TIP: Persistent command history
+
+If you want to persist your bash command history across container restart, simply add the following environment line in your `docker-compose.yaml` file:
```
Comment (JeremJR) [inline md/bcd_controller.md]: restarts?

File: md/upgrade_bcd.md [hunk 2]
```diff
@@ -66,7 +66,14 @@ cp -r bonita-continuous-delivery_2.0.0/terraform/your_stack_name bonita-continuo
...
 #### BCD controller
 
-As is described into the "Installation guide" from the [Getting started](getting_started.md) you will need to load the last version of `bcd-controller_<version>.tar.zip Docker image.
+As is described into the "Installation guide" from the [Getting started](getting_started.md) you will need to load the last version of `bcd-controller_<version>.tar.zip` Docker image.
+
+If you choosed to persist your docker console history ([see bcd controller](bcd_controller.md)) and want to keep it, then you should copy your `.bcd_bash_history` file from your old folder to your
+new `bonita-continuous-delivery_${varVersion}.0/ folder`
+
+```bash
+$ cp bonita-continuous-delivery_2.1.0/.bcd_bash_history bonita-continuous-delivery_${varVersion}.0/.bcd_bash_history
```
Comment (JeremJR) [inline md/upgrade_bcd.md]:
```suggestion
$ cp bonita-continuous-delivery_2.0.0/.bcd_bash_history bonita-continuous-delivery_2.0.2/.bcd_bash_history
```

File: md/bcd_controller.md (hunk 1 above)
Comment (JeremJR) [inline md/bcd_controller.md]: ``
```suggestion
If you want to persist your bash command history across container restarts, simply add the following environment line in your `docker-compose.yaml` file:
```

**Review by JeremJR — APPROVED** [review]

**Pull request merged**